examples/axual/33_application_access_grant_approvals.tf

```diff
@@ -0,0 +1,20 @@
+resource "axual_application_access_grant_approval" "dash_consume_logs_dev" {
+  application_access_grant = axual_application_access_grant.dash_consume_from_logs_in_dev.id
+}
+
+resource "axual_application_access_grant_approval" "dash_consume_logs_staging" {
+  application_access_grant = axual_application_access_grant.dash_consume_from_logs_in_staging.id
```

Comment:
When creating axual_application_access_grant_approval resource in non-auto environment i get this error:
<img width="1053" alt="image" src="https://github.com/Axual/terraform-provider-axual/assets/115068027/63de1381-d157-4271-b768-a1e81a42dcd1">

I am in the group that owns the application, auto-approval works, non auto approval doesn't

Reply:
I have failed to reproduce that.

Reply:
failed to reproduce as well